### ISSUE TITLE
RFC: Feature Ideas

### ISSUE THREAD

**Issue by the-tarin** [achuie/cookbook#2]
At a basic level

Must Have
- Read / Write Recipes
- Search bar for recipes
- Recipe preview bar on the side to toggle between recipes

Nice to Have
- Recipe Randomiser
- Tickbox for each ingredient/step

**Comment by achuie** [comment]
I’d say the preview sidebar would be a nice-to-have, it just adds more complicated rendering.

Also I think tickboxes will be easy, just a matter of parsing bullet points in markdown or whatever we decide as `<input type=“checkbox”>` so it could be a must-have

**Comment by the-tarin** [comment]
I agree with the suggestions @achuie. Will make an edit once we have come together and decided on our features.
